Makefile

```diff
@@ -100,12 +101,26 @@ bootstrap-flow:
 	cd build/flow && git checkout $(FLOW_COMMIT)
 
 test-flow:
-	node scripts/tests/flow/run_babel_parser_flow_tests.js
+	node scripts/tests/flow
 
 test-flow-ci: bootstrap test-flow
 
 test-flow-update-whitelist:
-	node scripts/tests/flow/run_babel_parser_flow_tests.js --update-whitelist
+	node scripts/tests/flow --update-whitelist
+
+bootstrap-typescript:
+	rm -rf ./build/typescript
+	mkdir -p ./build
+	git clone --branch=master --single-branch --shallow-since=2019-09-01 https://github.com/microsoft/TypeScript.git ./build/typescript
+	cd build/typescript && git checkout $(TYPESCRIPT_COMMIT)
+
+test-typescript:
+	node scripts/tests/typescript
+
+test-typescript-ci: bootstrap test-typescript
```

Comment:
It will fail on `make -j test-typescript-ci` since `test-typescript` depends on `bootstrap`. Could you rebase on master?